### PR TITLE
y-axis positioning update

### PR DESCRIPTION
**Pull request by jessipedia**
Eliminates overflow from the y-axis when oriented 'left', tested with rt-dash.

### CODE REVIEW

File: rickshaw.js [hunk 1]
```diff
@@ -1938,7 +1934,7 @@ Rickshaw.Graph.Axis.Y = Rickshaw.Class.create({
 
     if (this.orientation == 'left') {
       var berth = this.height * this.berthRate
```
Comment (evanemolo) [inline rickshaw.js]: is the `berth` declaration used anywhere else in `_drawAxis`'s scope? if not, remove. same goes for `this.berthRate`

**Comment by jessipedia** [comment]
Honestly, as far as I can tell nothing. All of the examples included in Rickshaw render the same without it, as does the rt-dash. `orientation: right` doesn't do any of the translation that berth was used to calculate, it was just being used to move `orientation: left` down. If you take out that, it looks exactly the same without any overflow. 

It does seem suspicious, but it doesn't seem like there's any functionality lost.

File: rickshaw.js (hunk 1 above)
Comment (evanemolo) [reply rickshaw.js]: edit - I should clarify: remove `this.berthRate` if it is not used anywhere else in the file, not just _drawAxis.

Comment (jessipedia) [reply rickshaw.js]: `this.berthRate` is still used to calculate axis height, but have removed unused `berth`s in `_drawAxis` and `setSize`

**Comment by jessipedia** [comment]
Went through the examples again, there is a change. Without this fix the axis positioning in `orientation: right` can be way off. 

Before:
screen shot 2019-02-14 at 11 42 09 am

After:
screen shot 2019-02-14 at 11 44 42 am

**Pull request merged**